entrypoint.sh

```diff
@@ -2,14 +2,9 @@
 
 set -e
 
-# Wait for the database to be ready
-until pg_isready -h metadata-db -p 5432 -U admin; do
-  echo "Waiting for the database to be ready..."
-  sleep 1
-done
+# Call create_db.sh script to create the database
+./create_db.sh
 
-# Run migrations and start the server
-python manage.py makemigrations
```

Comment:
Does this makemigrations line need to be in create_db.sh?

Reply:
No, we don't need to makemigrations since any pull request should handle it already. It was redundant before, and I just want to clean it up.